staker-rewards/lib/rewards-calculator.js

```diff
@@ -108,6 +121,21 @@ export default class RewardsCalculator {
     return new BigNumber(minimumStake)
   }
 
+  async isUndelegatingOperator(operatorAddress) {
+    const block = this.interval.startBlock
+
+    const tokenStaking = await this.context.contracts.TokenStaking.deployed()
+
+    const delegationInfo = await callWithRetry(
+      tokenStaking.methods.getDelegationInfo(operatorAddress),
+      block
```

Comment:
Can we use `this.interval.startBlock` here directly? I think it will be easier to read, especially that `block` is used only in one place.

Reply:
Sure